.github/workflows/full-test-windows.yml

```diff
@@ -0,0 +1,32 @@
+name: CI Build Windows Full
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+jobs:
+  check_commit:
+    uses: devonfw/ide/.github/workflows/check-commit.yml@master
+  
+  build:
+    needs: check_commit
+    if: ${{ needs.check_commit.outputs.should_run != 'false' }}
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Set up Shellcheck
+        run: choco install shellcheck
+      - name: Build project with Maven
+        run: |
+          export COMMAND_TEST=true
+          mvn -B -ntp install
+        
+        
```

Comment:
as you also removed the blank lines from `test-windows.yml` (👍 ) you could also do it here :)